tests/unit/test_referee_folder.py

```diff
@@ -1,204 +1,253 @@
 from onitu.referee.folder import Folder
 
 
-def check_size(size, length):
-        folder = Folder('folder', (), None,
-                        file_size={'min': size, 'max': size})
-        assert folder.min_size == length
-        assert folder.max_size == length
+class FakeLogger(object):
```

Comment:
Nope, unit tests are very confidential right now, so we don't really have any tools for them. I guess you could still use a regular logger from Logbook.